031-DevOpsWithGitHub/Coach/README.md

```diff
@@ -0,0 +1,60 @@
+# What The Hack - IoT Hack of the Century - Coach Guide
```

Comment:
I'm pretty sure this isn't the "IoT Hack of the Century" coach guide? ;) 

There's a couple of things in the template that didn't get updated.

Reply:
Ha!  You are right, I have updated